### ISSUE TITLE
--input=SingleFile.njk --output=. processes nothing

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#186]
![image](https://user-images.githubusercontent.com/39355/43551843-303a0bc0-95ae-11e8-9ed0-b28a0f79797e.png)

As you can see in the screenshot, `!./**` takes precedence over `./filename.njk`

**Comment by rendall** [comment]
This configuration in `.eleventy.js` will result in the output directory path `./false`:
``` 
dir: {
  input: "src",
  output: ""
 }
```
I'm mentioning it here because I suspect it's related, because this configuration
``` 
dir: {
  input: "src",
  output: "."
 }
```
results in no files processed.

**Comment by rendall** [comment]
Perhaps related to #232
``` 
dir: {
  input: "src",
  output: "/"
 }
```
results in the output directory being in the root directory!  That's unexpected.

**Comment by kleinfreund** [comment]
I expect the output directory configured with `output: "/"` to be the root directory. Same applies for `input: "/"`.

That said, I believe Eleventy shouldn’t allow neither the input nor the output directory to be outside of the project directory.

**Comment by zachleat** [comment]
@kleinfreund hmm good point that does seem potentially dangerous.

**Comment by kleinfreund** [comment]
I don’t think this is an issue. The input file location should rule out the output directory to be the same because people will eventually end up overwriting their own files. In general, the output directory should *never* be the Eleventy project directory.

**Comment by zachleat** [comment]
Fixed with #1129 but tests just added in 8c3377c01cd70b71b719924227b8f869f1b34400
